tests/resources/test_snippets.py

```diff
@@ -258,21 +258,37 @@ def _compare_snippets(snippet_db, snippet_api):
         '/snippets?title=snippet+%231&tag=tag_a&limit=1',
         headers={
             'Accept': 'application/json',
+            'Host': 'api.xsnippet.org',
+            'X-Forwarded-Proto': 'https',
         })
     assert resp.status == 200
     expected = [snippet]
     for snippet_db, snippet_api in zip(expected, await resp.json()):
         _compare_snippets(snippet_db, snippet_api)
+    # check that urls in Link preserve the additional query params
+    expected_link1 = (
+        '<https://api.xsnippet.org/snippets?title=snippet+%231&tag=tag_a&limit=1>; rel="first", '  # noqa
+        '<https://api.xsnippet.org/snippets?title=snippet+%231&tag=tag_a&limit=1&marker=3>; rel="next"'  # noqa
+    )
+    assert resp.headers['Link'] == expected_link1
 
     resp = await testapp.get(
         '/snippets?title=snippet+%231&tag=tag_a&limit=1&marker=3',
         headers={
             'Accept': 'application/json',
+            'Host': 'api.xsnippet.org',
+            'X-Forwarded-Proto': 'https',
         })
     assert resp.status == 200
     expected = [snippets[0]]
     for snippet_db, snippet_api in zip(expected, await resp.json()):
         _compare_snippets(snippet_db, snippet_api)
+    # check that urls in Link preserve the additional query params
```

Comment:
(minor) I used capitalize sentences in comments around the code. It'd be nice to follow this style if you don't mind. :) So you can ignore this one really, just FYI.

Reply:
Done!